src/main/java/edu/wisc/my/messages/service/ExpiredMessagePredicate.java

```diff
@@ -25,6 +25,15 @@ public ExpiredMessagePredicate(LocalDateTime asOfWhen) {
   public boolean test(Message message) {
     Validate.notNull(message);
 
+    try {
+      Validate.notNull(message.getFilter().getExpireDate());
+    } catch (Exception e) {
```

Comment:
I don't get it. Why is this `try-catch` block necessary?

The code following the block already handles `null` `filter` (considering the message unexpired) and exceptions in computing whether a provided expiration date makes the message expired (considering broken messages expired to fail closed.)